modules/storages/app/common/storages/peripherals/storage_interaction/one_drive/download_link_query.rb

```diff
@@ -78,8 +78,8 @@ def handle_errors(response)
             end
           end
 
-          def uri_path_for(file_id)
-            "/v1.0/drives/#{@storage.drive_id}/items/#{file_id}/content"
+          def url_for(file_id)
+            UrlBuilder.url(Util.drive_base_uri(@storage), "items", file_id, "content")
```

Comment:
A path was returned by the method before. Is it fine to return a uri now?

Reply:
probably overlooked. Will check, thanks